src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -43,24 +42,23 @@
     <Compile Include="System\ComponentModel\EnumConverter.cs" />
     <Compile Include="System\ComponentModel\EventDescriptor.cs" />
     <Compile Include="System\ComponentModel\EventDescriptorCollection.cs" />
-    <!--<Compile Include="System\ComponentModel\EventHandlerList.cs" />-->
+    <Compile Include="System\ComponentModel\EventHandlerList.cs" />
     <Compile Include="System\ComponentModel\ExtendedPropertyDescriptor.cs" />
-    <!--<Compile Include="System\ComponentModel\ExtenderProvidedPropertyAttribute.cs" />-->
     <Compile Include="System\ComponentModel\ExtenderProvidedPropertyAttribute.cs" />
```

Comment:
Weird - I just see the removal and no addition, although it's uncommented on the next line.